prac_09/taxi_simulator.py

```diff
@@ -43,8 +41,15 @@ def main():
             print_menu()
             menu_choice = input(">>> ").upper()
     print(f"Total trip cost: ${total_trip_cost:.2f}")
-    # TODO finish this print
-    print(f"Taxi's are now...")
+    print(f"Taxis are now")
+    print_taxis(taxis)
+
+
+def print_taxis(taxis):
+    taxi_number = 0
+    for taxi in taxis:
+        print(f"{taxi_number} - {taxi}")
+        taxi_number += 1
 
 
 def print_menu():
```

Comment:
All pretty much looks good to me 👍, only thing I see is the menu could just be a constant instead of a whole function, like this:
MENU = "Q)uit, C)hoose taxi, D)rive"